src/html/modules/report/about.html

```diff
@@ -0,0 +1,112 @@
+      </div>
+      <div class="container_column_40">
+        <select id="project" name="project">
+          <option value="australia">Australia</option>
+          <option value="canada">Canada</option>
+          <option value="usa">USA</option>
+        </select>
+      </div>
+    </div>
+    <!-- Site/Guardion -->
+    <div class="container_row">
+      <div class="container_column_25">
+        <label for="site-guardion">Site/Guardion</label>
+      </div>
+      <div class="container_column_40">
+        <select id="site-guardion" name="site-guardion">
+          <option value="australia">Australia</option>
+          <option value="canada">Canada</option>
+          <option value="usa">USA</option>
+        </select>
+      </div>
+    </div>
+    <!-- Incident Date -->
+    <div class="container_row">
+      <div class="container_column_25">
+        <label for="incident-date">Incident Date</label>
+      </div>
+      <div class="container_column_40">
+        <input type="date" id="date-report" name="date-report">
+      </div>
+    </div>
+    <!-- Incident Time -->
+    <div class="container_row">
+      <div class="container_column_25">
+        <label for="incident-time">Incident Time</label>
+      </div>
+      <div class="container_column_75">
+        <input type="time" id="time-report" name="time-report">
+        <p class="to-time">-</p>
+        <input type="time" id="time-report" name="time-report">
+      </div>
+    </div>
+    <!-- Incident Type -->
+    <div class="container_row">
+      <div class="container_column_25">
+        <label for="incident-type">Incident Type</label>
+      </div>
+      <div class="container_column_25">
+        <select id="incident-type" name="incident-type">
+          <option value="australia">Australia</option>
+          <option value="canada">Canada</option>
+          <option value="usa">USA</option>
+        </select>
+      </div>
+    </div>
+    <!-- Link (to Dashboard) -->
+    <div class="container_row">
+      <div class="container_column_25">
+        <label for="link-dashboard">Link (to Dashboard)</label>
+      </div>
+      <div class="container_column_75">
+        <input type="url" id="link-dashboard" name="link-dashboard">
+      </div>
+    </div>
+    <!-- Status -->
+    <div class="container_row">
+      <div class="container_column_25">
+        <label for="status">Status</label>
+      </div>
+      <div class="container_column_20">
+        <select id="status" name="status">
+          <option value="australia">Australia</option>
+          <option value="canada">Canada</option>
+          <option value="usa">USA</option>
+        </select>
+      </div>
+    </div>
+    <!-- Comments -->
+    <div class="container_row">
+      <div class="container_column_25">
+        <label for="comments">Comments</label>
+      </div>
+      <div class="container_column_75">
+        <textarea id="comments" name="comments" style="height:200px"></textarea>
```

Comment:
Please avoid using inline styles in this repo. Each html file has its own styles file where you can put all your styles code.